docs/ui/auth/fragments/web/authenticator.md

```diff
@@ -860,6 +860,35 @@ _App.vue_
 </template>
 ```
 
+<amplify-block-switcher>
+<amplify-block name="Vue 3">
+
+```js
+import {onAuthUIStateChange} from "@aws-amplify/ui-components";
+
+export default {
+  name: "AuthStateApp",
+  created() {
+    onAuthUIStateChange((authState, authData) => {
```

Comment:
Should this also have the `Auth.currentAuthenticatedUser().then((user) => this.user = user).catch(console.warn)` block?

Reply:
Based on our latest discussion, I'll work on resolving this from the library side.